tutorials/python-for-loops/index.mdx

```diff
@@ -0,0 +1,933 @@
+---
+meta:
+  title: Getting started with Python for loops, with exercises
+  description: This tutorial shows you how to get started with using for loops in Python, showing syntax, examples and practice exercises.
+content:
+  h1: Getting started with Python for loops
+  paragraph: This tutorial shows you how to get started with using for loops in Python, showing syntax, examples and practice exercises.
+tags: python for loop syntax examples statement
+categories:
+  - compute
+dates:
+  validation: 2023-06-28
+  posted: 2023-06-28
+---
+
+<Message type="requirement">
+If you're coming to this tutorial as a complete Python beginner, make sure you're familiar with the Python concepts covered in our [Python for complete beginners](/tutorials/get-started-python/) tutorial, and with [dictionaries and lists](/docs/tutorials/python-lists-dicts/).
+</Message>
+
+## Introduction
+
+So you know how to **create variables** in Python, as well as how to **create lists** and **dictionaries** to store multiple items in a single variable? Then you're ready to learn about **loops**.
+
+A **loop** is a piece of code that tells Python to repeat a specified set of actions multiple times. Loops are a fundamental part of most programming languages, and let you achieve a whole lot with just a few lines of code. Often, they **iterate** over collections of data (such as data held in dictionaries or lists) and carry out all sorts of operations on those collections.
+
+Why are loops useful? Let's look at a quick example:
```

Comment:
```suggestion
Why are loops useful? Let us look at a quick example:
```

not sure about this one though

Reply:
`Let's` is okay, the uncontracted form would be unnecessarily formal :)

Reply:
> not sure about this one though

`Let's ` is okay, the uncontracted form would be unnecessarily formal :)

Reply:
I agree, I'll leave it like it is